src/util_impl_linux.h

```diff
@@ -428,7 +429,26 @@ class CpuInfoLinux : public CpuInfo {
     numCores_[0] = numCores_[1] = sysconf(_SC_NPROCESSORS_ONLN);
   }
 
-  void setSysRegVal() { XBYAK_AARCH64_READ_SYSREG(cacheInfo_.midr_el1, MIDR_EL1); }
+  void setSysRegVal() {
+    /* Some Linux does not permit to access MIDR_EL1 register
+       from user program. The register value is disclosed
+       in /sys/devices/system/cpu/cpu?/regs/identification/midr_el1 */
+    char buf[64];
+    char *stop;
+
+    FILE *file = fopen(path_midr_el1, "r");
+    if (file == nullptr) {
+      throw Error(ERR_INTERNAL);
+      return;
+    }
+
+    if (fread(buf, sizeof(char), 64, file) == 0) {
```

Comment:
Hi @kawakami-k 
It's turned out that it leads to issue on Linux aarch64 in docker container. And during execution I have:
```
(gdb) c
Continuing.
bad err=11 in Xbyak::Error
```
Because current line cannot read anything from file. This is because `/sys/devices/system/cpu/cpu?/regs/identification/midr_el1` is empty:
```
% docker run -it --rm ubuntu:20.04
root@0ff70ef278d2:/# ls /sys/devices/system/cpu/cpu0/regs/identification/midr_el1
/sys/devices/system/cpu/cpu0/regs/identification/midr_el1
root@0ff70ef278d2:/# cat /sys/devices/system/cpu/cpu0/regs/identification/midr_el1
root@0ff70ef278d2:/# 
```